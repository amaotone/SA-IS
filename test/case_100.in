14159265358979323846
26433832795028841971
69399375105820974944
59230781640628620899
86280348253421170679
