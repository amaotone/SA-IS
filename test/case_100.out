63 70 82 35
